docs/mmctl.rst

```diff
@@ -16,6 +16,7 @@ Options
 
 ::
 
+      --config string                path to search for '.mmctl' configuration file (default "/home/ferdinand/.config/mmctl")
```

Comment:
The default path depends on who is running the command. Can you change this to e.g. `$HOME/.config/mmctl`?

Reply:
Sorry, my bad, I should have seen this!